### PR TITLE
New operator

### PR DESCRIPTION
**Pull request by matzew**

### CODE REVIEW

File: openshift/e2e-tests-openshift.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@ source $(dirname $0)/release/resolve.sh
 
 set -x
 
-readonly SERVING_VERSION=v0.7.1
+readonly SERVING_VERSION=v0.6.0
```
Comment (syedriko) [inline openshift/e2e-tests-openshift.sh]: Was this supposed to be 0.8.0?

File: openshift/e2e-tests-openshift.sh [hunk 2]
```diff
@@ -52,12 +49,12 @@ function install_strimzi(){
 function install_knative_serving(){
   header "Installing Knative Serving"
 
-  create_knative_namespace serving
+  # create_knative_namespace serving
```
Comment (syedriko) [inline openshift/e2e-tests-openshift.sh]: May be just delete this line?

File: openshift/e2e-tests-openshift.sh [hunk 3]
```diff
@@ -136,18 +122,18 @@ function deploy_knative_operator(){
 function install_knative_eventing(){
   header "Installing Knative Eventing"
 
-  create_knative_namespace eventing
-
+  # create_knative_namespace eventing
```
Comment (syedriko) [inline openshift/e2e-tests-openshift.sh]: Just delete this line may be?

File: openshift/e2e-tests-openshift.sh [hunk 4]
```diff
@@ -136,18 +122,18 @@ function deploy_knative_operator(){
...
   echo ">> Patching Knative Eventing CatalogSource to reference CI produced images"
   CURRENT_GIT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
   RELEASE_YAML="https://raw.githubusercontent.com/openshift/knative-eventing/${CURRENT_GIT_BRANCH}/openshift/release/knative-eventing-ci.yaml"
   sed "s|--filename=.*|--filename=${RELEASE_YAML}|"  openshift/olm/knative-eventing.catalogsource.yaml > knative-eventing.catalogsource-ci.yaml
 
   # Install CatalogSources in OLM namespace
-  oc apply -n $OLM_NAMESPACE_EVENTING -f knative-eventing.catalogsource-ci.yaml
-  timeout_non_zero 900 '[[ $(oc get pods -n $OLM_NAMESPACE_EVENTING | grep -c knative-eventing) -eq 0 ]]' || return 1
-  wait_until_pods_running $OLM_NAMESPACE_EVENTING
+#  oc apply -n $OLM_NAMESPACE -f knative-eventing.catalogsource-ci.yaml
```
Comment (syedriko) [inline openshift/e2e-tests-openshift.sh]: And this one may be just delete it?

File: openshift/release/generate-release.sh [hunk 5]
```diff
@@ -15,15 +14,15 @@ else
 fi
 
 # the core parts
-resolve_resources config/ $output_file $image_prefix $release
+resolve_resources config/ $output_file $image_prefix $tag
 
 # in-memory-channel (ccp)
 # TODO: remove in 0.8.0 - because the deprecated CCP is removed by than
```
Comment (syedriko) [inline openshift/release/generate-release.sh]: It's 0.8.0 - act on the comment or remove it?

File: openshift/e2e-tests-openshift.sh (hunk 1 above)
Comment (matzew) [reply openshift/e2e-tests-openshift.sh]: no - that's to make this build work.

soon will be TP1 (0.7.1)

File: openshift/e2e-tests-openshift.sh (hunk 2 above)
Comment (matzew) [reply openshift/e2e-tests-openshift.sh]: Nope, needed with TP1 - this is to get the 0.8.0 build going ;-)

File: openshift/e2e-tests-openshift.sh (hunk 4 above)
Comment (matzew) [reply openshift/e2e-tests-openshift.sh]: nope, libe above

File: openshift/release/generate-release.sh (hunk 5 above)
Comment (matzew) [reply openshift/release/generate-release.sh]: that was always wrong, and did cause wrong `:ci` tags

**Pull request merged**